plugnpy/metric.py

```diff
@@ -63,8 +83,11 @@ def __init__(
         self.message = None
         self.convert_metric = convert_metric
 
-        bytes_conversion_factor = 1000 if si_bytes_conversion else 1024
-        self.conversion_factor = bytes_conversion_factor if self.unit == self.BYTES_UNIT else 1000
+        self.si_bytes_conversion = si_bytes_conversion
+        bytes_conversion_factor = Metric.SI_UNIT_FACTOR if si_bytes_conversion else Metric.IEC_UNIT_FACTOR
+        self.conversion_factor = bytes_conversion_factor if self.unit == Metric.UNIT_BYTES else Metric.SI_UNIT_FACTOR
+
```

Comment:
Could we do:
`if self.unit == Metric.UNIT_BYTES and not si_bytes_conversion:`
`     self.conversion_factor = Metric.IEC_UNIT_FACTOR`
`else Metric.SI_UNIT_FACTOR`